src/app/layouts/header/export-popup/export-popup.component.ts

```diff
@@ -73,6 +76,31 @@ export class ExportPopupComponent implements OnInit, OnDestroy {
         this.popup.hide();
     }
 
+    exportZip(): void {
+        const zip = new JSZip();
+        const fontExtension=['ttf', 'woff', 'woff2'];
+        let choice: string;
+        const fileName = 'dx.' + this.importService.getThemeName() + '.' + this.schemeName + '.css';
+
+        if(this.importService.getThemeName() === 'generic') {
+            choice = 'content/css/icons/dxicons';
+        } else {
+            choice = 'content/css/icons/dxiconsmaterial';
+        }
+
+        zip.file(fileName, this.importService.exportCss(this.schemeName, this.makeSwatch));
+
+        fontExtension.forEach( function (extension) {
+            const filename = choice + '.' + extension;
+            zip.file(filename, JSZipUtils.getBinaryContent(filename));
+        });
+        zip.generateAsync({type: 'blob'})
+                    .then(function(content) {
+                        saveAs(content, fileName + '.zip');
```

Comment:
If we use fileName variable here then the archive has `dx.generic.light.css.zip` name. I think '.css' is redundant in this case.